apps/admin/package.json

```diff
@@ -46,10 +46,11 @@
     "@types/react-dom": "^18.2.7",
     "@types/react-syntax-highlighter": "^15.5.7",
     "autoprefixer": "^10.4.14",
+    "dnd-core": "^16.0.1",
     "dotenv": "^16.3.1",
-    "eslint": "^8.45.0",
-    "eslint-config-next": "^13.4.10",
-    "eslint-plugin-import": "^2.27.5",
+    "eslint": "^8.57.0",
```

Comment:
I would have liked to bump ESLint to 9. But Vercel has been slow to update for ESLint 9 and flat config:

- [x] https://github.com/vercel/next.js/issues/64409
- [ ] https://github.com/vercel/style-guide/issues/73

Reply:
ooo looks like theres a pr open: https://github.com/vercel/style-guide/pull/114